firefly_client/firefly_client.py

```diff
@@ -704,6 +704,10 @@ def show_table(self, file_on_server=None, tbl_id=None, title=None, page_size=100
                 if table shows units for the columns.
             **showFilters** : `bool`
                 if table shows filter button
+        table_index : `int`, optional
+            The table to be shown in case `file_on_server` contains multiple tables. It is extension number for
+            a fits file or table index for a votable file. In default, table of extension 1 from a fits file or
```

Comment:
"a FITS file or the table index for a VOTable file. If unspecified, the server will fetch extension 1 from a FITS file or"